compiler/control/OMROptions.cpp

```diff
@@ -1982,6 +1982,11 @@ OMR::Options::jitLatePostProcess(TR::OptionSet *optionSet, void * jitConfig)
    // OSR is not available for ARM yet
    self()->setOption(TR_EnableOSR, false);
    self()->setOption(TR_EnableOSROnGuardFailure, false);
+   self()->setOption(TR_DisableNextGenHCR);
+#endif
+
+#ifndef J9_PROJECT_SPECIFIC
+   self()->setOption(TR_DisableNextGenHCR);
```

Comment:
Is it certain that `OMR::Options::jitLatePostProcess` will run before `OMR::Compilation::Compilation` constructor which queries this option and make some decisions based on it?